pulp_2to3_migration/app/plugin/__init__.py

```diff
@@ -1,8 +1,19 @@
+from gettext import gettext as _
+from collections import namedtuple
+import logging
 import pkg_resources
 
+_logger = logging.getLogger(__name__)
+
 # { plugin_name: PluginMigratorClass }
 PLUGIN_MIGRATORS = {}
+MissingMigrator = namedtuple('MissingMigrator', 'pulp3_plugin')
 
 if not PLUGIN_MIGRATORS:
     for entry_point in pkg_resources.iter_entry_points(group='migrators'):
-        PLUGIN_MIGRATORS[entry_point.name] = entry_point.load()
+        try:
+            PLUGIN_MIGRATORS[entry_point.name] = entry_point.load()
+        except ModuleNotFoundError as exc:
+            _logger.warning(_('%s') % exc.msg)
```

Comment:
I'm unsure if it warrants a warning, since not having all the plugins supported by the migrators is a fairly normal occurrence.  Maybe log as INFO?

Reply:
I was unsure if I should go with info or warning, but as serializer [validates if plugin is installed](https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/serializers.py#L93-L96), I think we can change it to info

Reply:
since we are sending this to info shall we write a more user friendly error message? "Plugin X is not installed in pulp3 therefore it will not be migrated from pulp2" (something along the lines)

Reply:
done:
```
pulp: pulp_2to3_migration.app.plugin:INFO: Plugin pulp_container is not installed in pulp3 therefore it will not be migrated from pulp2
```